c10/core/DispatchKey.h

```diff
@@ -439,6 +439,8 @@ enum class DispatchKey : uint16_t {
   Autograd,
   CompositeImplicitAutograd, // registered at
   // build/aten/src/ATen/RegisterCompositeImplicitAutograd.cpp
+  CompositeImplicitAutogradNestedTensor, // registered at
```

Comment:
> hmmm I think you'll need to make internal fbcode changes, since this adds a new codegen'd file.
> 
> I would fbgs search for `RegisterCompositeImplicitAutograd.cpp` to see where that file gets hardcoded into the build system, to see where you need to add it